skr/go.mod

```diff
@@ -5,28 +5,37 @@ go 1.18
 require (
 	github.com/go-logr/logr v1.2.3
 	github.com/kyma-project/kyma-watcher/kcp v0.0.0-20220801122243-5ce523a796aa
+	github.com/kyma-project/manifest-operator/operator v0.0.0-20220722124726-ba2878448c7d
 	github.com/onsi/ginkgo v1.16.5
```

Comment:
If you have v2 in your dependencies, v1 is unnecessary and you can remove it

Reply:
It's still used in the previous skr-watcher project. When the previous skr-watcher is cleaned, this will be gone too.